crates/meroctl/src/cli/bootstrap/start.rs

```diff
@@ -313,11 +313,11 @@ impl StartBootstrapCommand {
         let (context_id, public_key) = create_context(
             environment,
             &client,
-            multiaddr,
+            connection,
             None,
             application_id,
             None,
-            &config.identity,
+            auth_key,
```

Comment:
since `auth_key` is now in connection, you shouldn't need to pass both of these, just use it from the connection